package.json

```diff
@@ -160,9 +160,10 @@
     "omelette": "^0.4.12",
     "semver": "^7.3.2",
     "strip-json-comments": "^3.1.1",
-    "typescript": "^3.8.3",
+    "typescript": "^4.1.3",
```

Comment:
I think this change is not intended, right? We're already on typescript@4.1.3 so there is no need to update it.